src/Algebra/Construct/NaturalChoice/Max.agda

```diff
@@ -0,0 +1,28 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- The max operator derived from an arbitrary total order
+------------------------------------------------------------------------
+
+open import Relation.Binary
+
+module Algebra.Construct.NaturalChoice.Max
+  {a ℓ₁ ℓ₂} (totalOrder : TotalOrder a ℓ₁ ℓ₂) where
+
+import Relation.Binary.Construct.Converse as Converse
+
+------------------------------------------------------------------------
+-- Max is just min with a flipped order
+
+open import Algebra.Construct.NaturalChoice.Min
+  using
```

Comment:
Did you mean to apply `Algebra.Construct.NaturalChoice.Min` to `Converse.totalOrder totalOrder`?

Reply:
Haha yes, I wonder where that went!